packages/twenty-server/src/engine/core-modules/two-factor-authentication/entities/two-factor-authentication-method.entity.ts

```diff
@@ -25,16 +29,26 @@ export class TwoFactorMethod {
   userWorkspaceId: string;
 
   @Field(() => UserWorkspace)
-  @ManyToOne(
+  @OneToOne(
```

Comment:
Are you sure about this change? What if you could have TOTP + Backup codes as 2 method registered for a same userworkspace? I thought this was the point of your changes introducing strategy everywhere